src/jit/flowgraph.cpp

```diff
@@ -25694,3 +25694,211 @@ bool Compiler::fgUseThrowHelperBlocks()
 {
     return !opts.compDbgCode;
 }
+
+//------------------------------------------------------------------------
+// fgTailMergeThrows: Tail merge throw blocks and blocks with no return calls.
+//
+// Notes:
+//    Scans the flow graph for throw blocks and blocks with no return calls
+//    that can be merged, and opportunistically merges them.
+//
+//    Does not handle throws yet as the analysis is more costly and less
+//    likely to pay off.
```

Comment:
I think it would be useful to give a code example here in the header comment of the pattern you are looking for, and describe why this is valuable.

Reply:
Done.